NEWS.md

```diff
@@ -1,6 +1,12 @@
 
 # vctrs (development version)
 
+* Double-dispatch methods for `vec_ptype2()` are now easier to
+  implement. vctrs takes care of implementing the default and
+  unspecified methods. Methods no longer have to worry about order of
+  dispatch, for instance it is no longer needed to supply a second
+  argument as in `UseMethod("vec_ptype2.mytype", y)`.
```

Comment:
Mention explicitly that `NextMethod()` doesn't work, and it never worked correctly?